packages/td-tools/src/util/asset-interface-description.ts

```diff
@@ -818,16 +815,17 @@ export class AssetInterfaceDescriptionUtil {
             for (const [key, value] of smInformation.events.entries()) {
                 logInfo("Event " + key + " = " + value);
 
-                thing.events[key] = {};
-                thing.events[key].forms = [];
-
-                for (const vi of value) {
+                const forms = value.map((vi) => {
```

Comment:
Why are we changing the paradigm here? didn't we use `for of` for the other affordances above?